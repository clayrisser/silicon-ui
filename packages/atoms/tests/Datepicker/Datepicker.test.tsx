import React from 'react';
import renderer from 'react-test-renderer';
// import { render, fireEvent } from '@testing-library/react';
import DatePicker from '../../src/Datepicker';

describe('<DatePicker />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<DatePicker />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
