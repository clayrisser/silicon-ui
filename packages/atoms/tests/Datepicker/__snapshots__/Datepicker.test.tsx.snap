// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DatePicker /> renders correctly 1`] = `
<input
  className="css-119ltm9"
  fontFamily="body"
  fontSize={0}
  fontWeight="body"
  onChange={[Function]}
  type="date"
  value=""
  width="100%"
/>
`;
